sdks/python/apache_beam/yaml/yaml_mapping.py

```diff
@@ -515,7 +515,7 @@ def normalize_fields(pcoll, fields, drop=(), append=False, language='generic'):
 
   if append:
     return input_schema, {
-        **{name: name
+        **{name: f'`{name}`' if language == 'sql' else name
```

Comment:
looks like these issues should be easily caught with some tests. So can we add more tests?

Reply:
Added some tests